addons/website_forum/models/forum_tag.py

```diff
@@ -26,10 +32,41 @@ def _compute_posts_count(self):
         for tag in self:
             tag.posts_count = len(tag.post_ids)  # state filter is in field domain
 
+    @api.depends("forum_id", "forum_id.name", "name")
+    def _compute_website_url(self):
+        for tag in self:
+            tag.website_url = f'/forum/{slug(tag.forum_id)}/tag/{slug(tag)}/questions'
+
     @api.model_create_multi
     def create(self, vals_list):
         for vals in vals_list:
             forum = self.env['forum.forum'].browse(vals.get('forum_id'))
             if self.env.user.karma < forum.karma_tag_create and not self.env.is_admin():
                 raise AccessError(_('%d karma required to create a new Tag.', forum.karma_tag_create))
         return super(Tags, self.with_context(mail_create_nolog=True, mail_create_nosubscribe=True)).create(vals_list)
+
+    # ----------------------------------------------------------------------
+    # WEBSITE
+    # ----------------------------------------------------------------------
+
+    @api.model
+    def _search_get_detail(self, website, order, options):
+        search_fields = ['name']
+        fetch_fields = ['id', 'name', 'website_url']
+        mapping = {
+            'name': {'name': 'name', 'type': 'text', 'match': True},
+            'website_url': {'name': 'website_url', 'type': 'text', 'truncate': False},
+        }
+        base_domain = []
+        if forum := options.get("forum"):
+            forum_ids = (unslug(forum)[1], ) if isinstance(forum, str) else forum.ids
+            base_domain = [[('forum_id', 'in', forum_ids)]]
```

Comment:
why not that ? (doesn't look like a valid domain)
```suggestion
            base_domain = [('forum_id', 'in', forum_ids)]
```

Reply:
`base_domain`expects... a list of domains.
From the mixin:
```py
    def _search_get_detail(self, website, order, options):
        """...
        - base_domain: list of domains within which to perform the search
        ...
        """
```